src/sentry/runner/commands/devserver.py

```diff
@@ -17,10 +17,11 @@
 @click.option('--reload/--no-reload', default=True, help='Autoreloading of python files.')
 @click.option('--watchers/--no-watchers', default=True, help='Watch static files and recompile on changes.')
 @click.option('--workers/--no-workers', default=False, help='Run asynchronous workers.')
+@click.option('--refresh/--no-refresh', default=False, help='Automatic browser refreshing on webpack builds')
```

Comment:
Can we rename this `--livereload` ? That seems much more fitting than `--refresh`. It's liteally a live reload plugin.